.storybook/decorators/storyDarkModeDecorator.js

```diff
@@ -1,11 +1,32 @@
-import { withThemeByDataAttribute } from "@storybook/addon-themes";
-
-export const storyDarkModeDecorator = (darkClass, lightClass) =>
-  withThemeByDataAttribute({
-    attributeName: "class",
-    defaultTheme: "light",
-    themes: {
-      light: lightClass,
-      dark: darkClass,
-    },
-  });
+import { DecoratorHelpers } from "@storybook/addon-themes";
+import { vuelessConfig } from "../../src/utils/ui.ts";
+import { setTheme } from "../../src/index.ts";
+
+const { initializeThemeState, pluckThemeFromContext } = DecoratorHelpers;
+
+export const storyDarkModeDecorator = () => {
+  const prefersColorSchemeDark = window.matchMedia("(prefers-color-scheme: dark)");
+  const preferScheme = prefersColorSchemeDark.matches ? "dark" : "light";
+  const cachedColorMode = localStorage.getItem("vl-color-mode") || preferScheme;
+
+  initializeThemeState(["light", "dark"], cachedColorMode);
+
+  return (story, context) => {
+    const theme = pluckThemeFromContext(context);
+
+    setTheme({
```

Comment:
you already reading `vuelessConfig` inside the `setTheme` so there is no reason to pass this params again.

Please fix it in all similar places.

Reply:
Please fix it in all places.

![image](https://github.com/user-attachments/assets/c8826b63-307d-421e-8f8d-c4d64f2dde12)